ui/src/components/identities/IdentityForm.tsx

```diff
@@ -82,13 +114,20 @@ export function IdentityForm({
     return aborter;
   }, [fetchNetworks]);
 
-  if (hasAsyncTaskFailed(supportedNetworks)) {
+  if (
+    hasAsyncTaskFailed(supportedNetworks) ||
+    (isAsyncTaskDataAvailable(supportedNetworks) && !supportedNetworks.data.length)
+  ) {
+    const errorMessage = hasAsyncTaskFailed(supportedNetworks)
+      ? supportedNetworks.error.message
+      : "";
+
     return (
       <Card className="centered">
```

Comment:
We don't want/need these double Card wrappers:

<img width="738" alt="image" src="https://github.com/user-attachments/assets/e3df96cd-7d5d-46a7-97f9-d37b94cdf401">

<img width="741" alt="image" src="https://github.com/user-attachments/assets/63019686-4bf8-47fc-a598-7ba2b28fc032">

Let's return just the `ErrorResult` and `LoadingResult` components.